### ISSUE TITLE
Login mistakenly asking for Two Factor code

### ISSUE THREAD

**Issue by ViGrey** [tootsuite/mastodon#1310]
I have 2 accounts on mastodon.social and if I try logging into either account, the login page will ask me for a Two Factor code even though I have not set up Two Factor Authentication on either account.  This problem occurs on all browsers I try on all operating systems and on all networks I try.  I have managed to get into one of my accounts after clearing my cache and cookies and logging in a few hundred times, but I have only managed to successfully do that twice. 

### _Likely useless but possibly useful information_
Occasionally, when I clear out my cookies and cache and then try logging in, the information on the page asking me for a Two Factor Code will be in French.

While I can guarantee I did not set up Two Factor Authentication on any of my accounts, I cannot guarantee I didn't click on the Two Factor Authentication setup option.

**Comment by ViGrey** [comment]
Quick update.  I was able to log into one of the accounts a single time within the last 10 minutes, then after logging out, I had the problem again.  I was not able to log into the other account.  No javascript errors are happening.

**Comment by ViGrey** [comment]
Another update.  After randomly deleting cookies enough times, I was able to log in once again on the account that I got into last time.  The other account has not been able to be logged into since I started having this problem.  For almost every case where I'm finally able to log in, I usually have to try and randomly trigger a login error (Like the message asking if cookies are disabled) first, then go back to the login page without resending the POST values, then try logging in.  In those cases, it only seems to work very rarely.  I'm a little afraid to log out, so I'm just going to try and stay logged in for right now.

**Comment by shawnyeager** [comment]
I'm having the same problem. I never setup 2FA, but I'm being challenged on each attempted login to mastodon.social.

**Comment by alberto-santini** [comment]
I am having the same problem and I am currently locked out of oulipo.social

**Comment by ViGrey** [comment]
Just confirming that the 2FA problem still exists for both of my accounts on mastodon.social.

**Comment by matthewmcvickar** [comment]
I have never been able to log in to mastodon.social because of this issue. Is there no way at all to recover my account?

**Comment by Gargron** [comment]
@matthewmcvickar @ViGrey Sure is. E-mail your admin. In your cases that's me, my e-mail is eugen@zeonfederated.com, you can always find a contact e-mail on the /about/more page.

I suspect a lot of people in this thread is people who accidentally turned on 2FA back in the period of Mastodon where turning on 2FA did not require any confirmation, just a click of a button.

**Comment by ViGrey** [comment]
@Gargron Thank you for replying.  It's strange that I was able to get into one of my accounts that has this problem a few times, although it was exceptionally inconsistent.  If it never let me into any of the accounts that are suffering from this problem after the problem started, things would make more sense to me :P

**Comment by oz** [comment]
This _seem_ related, so I'm resisting opening a new issue:

1. I enabled 2FA long ago (with recovery codes, etc).
2. I tried to login today on a new browser, but my instance (mastodon.social) did not accept the 2FA code.
3. I disabled 2FA security from another session -- using a recovery code worked, because 2FA codes were rejected here too.
4. Login works normally! :smile: 
5. Trying to activate 2FA from settings now seem to fail. :sob:
